src/main/docker/Dockerfile.agent

```diff
@@ -6,6 +6,7 @@
 FROM fabric8/java-alpine-openjdk11-jre
 
 # add mvn, npm, nodejs, hugo
+RUN apk add -U --upgrade nghttp2
```

Comment:
Isn't it enough to add nghttp2 to the apk add command from the line below?

Reply:
I can try w/o the upgrade flag and check, but the bug resolution on the Alpine GL was to add the upgrade flag as well

Reply:
As fabric8 is discontinued (http://fabric8.io), I guess we may also want to start thinking switching the parent image to a recent alpine and install openjdk11/mvn/... ourselves instead. That would "solve" the issue.

Reply:
The build failed with the same message as before, so the upgrade is necessary for the nghttp2 item. Do we want to revert that little change, or look at shifting away from fabric for this fix?

Reply:
For future proofing, I guess it would be better to move away from fabric8. 

From https://hub.docker.com/r/fabric8/java-alpine-openjdk11-jre/dockerfile, it seems that just switching to

```Dockerfile
FROM alpine:3.12 
``` 
(3.12 is latest)

and adding `openjdk11-jre` to the `apk add` list should be enough.

Thanks for digging into this.

Reply:
Jenkins failure due to 502 getting Quarkus assets. Rebuilding now

Reply:
By using `alpine:3.12` (which gets updated on a regular basis), you should not need the separate 
```
RUN apk add -U --upgrade nghttp2
```

Reply:
Good point, done :)